test/gerber-parser_drill_test.js

```diff
@@ -379,6 +379,28 @@ describe('gerber parser with gerber files', function() {
 
       p.write('X1Y1\n')
     })
+
+    it('should warn / detect trailing zeros if possible', function(done) {
+      var warnings = []
+
+      p.on('warning', function(w) {
+        warnings.push(w)
+      })
+
+      p.on('finish', function() {
+        expect(warnings[0].message).to.match(/assuming trailing/)
+        expect(warnings[1].message).to.match(/detected leading/)
+        expect(p.format.zero).to.equal('L')
+        done()
+      })
+
+      p.write(';FILE_FORMAT=2:4\n')
```

Comment:
I was just trying to _only_ have the warnings I care about otherwise I have to make how many other warnings there are part of the test, or do something more complicated.